react-basic-practice/src/components/CustomerView/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { Box, FormLabel, Grid, GridItem, Text } from "@chakra-ui/react";
+import { ICustomer } from "@types";
+import TableFinance from "../TableFinance";
+import TableStatus from "../TableStatus";
+
+interface ViewCustomerProps {
+  data: ICustomer;
+}
+
+const FormView: React.FC<ViewCustomerProps> = ({ data }) => {
+  return (
+    <Grid templateColumns="repeat(2, 1fr)" rowGap="50px">
+      <GridItem colSpan={1}>
+        <Box>
+          <FormLabel>Name</FormLabel>
```

Comment:
We don't have `Form` anymore. Why we use `FormLabel` here. Pls update

Reply:
i done